presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -815,6 +846,10 @@ example uses the earliest snapshot ID: ``2423571386296047175``
       testBranch | BRANCH | 3374797416068698476 | NULL                    | NULL                  | NULL
       testTag    | TAG    | 4686954189838128572 | 10                      | NULL                  | NULL
 
+Presto C++ Support
+^^^^^^^^^^^^^^^^^^
+
+All above extra hidden metadata tables are supported in Presto C++.
```

Comment:
Just want to double confirm that you have verified they are working, right?

Reply:
Apart from`$changelog` everything works, `$changelog` throws an error which I missed adding here. I opened an issue for the same. https://github.com/prestodb/presto/issues/24816